config/locales/en.yml

```diff
@@ -89,11 +89,19 @@ en:
           attributes:
             expires_at:
               inclusion: "must be in the future"
+        org:
```

Comment:
This needs to be updated because of the rename to organization.

```suggestion
        organization:
```

Reply:
thank you!